action.yml

```diff
@@ -53,20 +53,50 @@ inputs:
 runs:
   using: 'composite'
   steps:
+    - name: Check if review is requested
+      shell: bash
+      env:
+        PR_TITLE: ${{ github.event.pull_request.title }}
+        PATTERN: \\b((?:no|skip)-(?:review|otter|otterai)|otter-(?:no|bye|restricted))(?:,((?:no|skip)-(?:review|otter|otterai)|otter-(?:no|bye|restricted)))*\\b
+      run: |
+        if echo "$PR_TITLE" | grep -Eiq "$PATTERN"; then
+          echo "🦦 No review requested, skipping code review"
+          echo "code_review_requested=false" >> $GITHUB_OUTPUT
+        else
+          echo "🔍 Code review requested"
+          echo "code_review_requested=true" >> $GITHUB_OUTPUT
+        fi
+
+    - name: Check if PR is merged
+      shell: bash
+      env:
+        PR_STATE: ${{ github.event.pull_request.state }}
+        PATTERN: \\b(merged|closed)\\b
```

Comment:
🔍 Ensure that the condition for checking if the PR is merged is robust against unexpected states.